griptape/engines/query/vector_query_engine.py

```diff
@@ -36,25 +37,25 @@ def query(
             if isinstance(artifact, TextArtifact)
         ]
         text_segments = []
-        message = ""
+        user_message = ""
 
         for artifact in artifacts:
             text_segments.append(artifact.value)
 
-            message = self.template_generator.render(
+            user_message = self.user_template_generator.render(
                 metadata=metadata,
                 query=query,
                 text_segments=text_segments,
```

Comment:
Segments can be moved into the system message as well, so the LLM is more "grounded."